pkg/apis/crd/v1alpha2/types.go

```diff
@@ -543,3 +543,88 @@ type TrafficControlList struct {
 
 	Items []TrafficControl `json:"items"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// BGPPolicy defines BGP configuration applied to Nodes.
+type BGPPolicy struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard metadata of the object.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec BGPPolicySpec `json:"spec"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type BGPPolicyList struct {
+	metav1.TypeMeta `json:",inline"`
+	// +optional
+	metav1.ListMeta `json:"metadata,omitempty"`
+
+	Items []BGPPolicy `json:"items"`
+}
+
+// BGPPolicySpec defines the specification for a BGP policy.
+type BGPPolicySpec struct {
+	// NodeSelector is utilized to select Nodes to which the policy is applied.
+	NodeSelector *metav1.LabelSelector `json:"nodeSelector"`
+
+	// LocalASN is the AS number used by BGP protocol.
+	LocalASN int32 `json:"localASN"`
+
+	// ListenPort is the port on which BGP protocol listen.
```

Comment:
```suggestion
	// ListenPort is the port on which the BGP protocol listens.
```
Could you indicate the default value as part of the comment as well, even though it is probably well-known?

Reply:
Done